### DISCUSSION TITLE
Why use TraceContext.traceId() but get Ignored_Trace?

### DISCUSSION THREAD

**Discussion by gaiwey** [apache/skywalking#9078] [Q&A] [answered]
apm-toolkit-trace 8.9.0
Why use TraceContext.traceId() but get Ignored_Trace?
![image](https://user-images.githubusercontent.com/21032881/168751270-6b2f6b68-3ece-4447-be12-ed80f08ba06b.png)

**Comment by gaiwey** [comment, marked as answer]
The answer is here：https://github.com/apache/skywalking/issues/4734

**Reply by Bigbobc** [reply]
How did you finally solve this problem?
